ide-config/eclipse.importorder

```diff
@@ -0,0 +1,6 @@
+#Organize Import Order
```

Comment:
How and where is this file used?

Reply:
It's setting file for Eclipse to be comliat with `impsort-maven-plugin` configuration.

It's just making stuff equivalent between mvn based formatting and Eclipse based.